### PR TITLE
Bug fix for the overflow problem on screens with viewport width of 360px and below  

### PR DESCRIPTION
**Pull request by bilelBoulhia**
issue link : https://github.com/Heroic-Games-Launcher/heroic-website/issues/27

to fix the problem  : 
1. changed main  display to flex for more responsiveness and made the overflow hidden
2. removed hardcoded sizes for the feature image and changed it to 100%
3. added inline padding for the section title .
(note : i installed markdoc dependency  and then removed it )

I have put a comment under each change to make them clear

### CODE REVIEW

**Comment by bilelBoulhia** [comment]
@flavioislima can you please check this PR please (if you have the time for it of course)

**Comment by flavioislima** [comment]
Thanks for fixing the issue. Can you add the .idea folder to .gitignore as well?

**Comment by bilelBoulhia** [comment]
> Thanks for fixing the issue. Can you add the .idea folder to .gitignore as well?

yeah sure

File: pages/index.tsx [hunk 1]
```diff
@@ -24,6 +24,10 @@ const transition = { duration: 0.5 }
 const Home: NextPage = () => {
   return (
     <>
+
```
Comment (flavioislima) [inline pages/index.tsx:27]: Why those changes in spacing?
Can you use the auto format feature of the IDE to use prettier to format all files?

File: package-lock.json
[the remark this answers is not in the document]
Comment (bilelBoulhia) [reply package-lock.json:1]: @flavioislima  i have removed it

File: pages/index.tsx (hunk 1 above)
Comment (bilelBoulhia) [reply pages/index.tsx]: i've run prettier reformatting on pages / styles / components and config files , i hope that's okay

**Pull request merged**